src/styles/styles.js

```diff
@@ -698,6 +698,10 @@ const styles = {
         borderRadius: variables.componentBorderRadiusNormal,
     },
 
+    textInputScrollPadding: {
+        scrollPadding: '23px 0 0 0',
```

Comment:
Does this create a warning on the native Platform as this style prop is not supported there? 

🙃 ![image](https://user-images.githubusercontent.com/24370807/158201563-8464b0c9-d087-4ff0-a5cd-45c03510699b.png)

Reply:
What is the better way to check is native or not? Using `getPlatform()` or `getOperatingSystem()`?

Reply:
None of them. Please take a look at styling docs and let me know if you have better suggestions.